tests/acceptance/01_vars/01_basic/nested_expansion_namespaced_variables.cf

```diff
@@ -10,8 +10,8 @@ body common control
 bundle agent init
 {
   meta:
-      "test_soft_fail" string => "any",
-        meta => { "redmine6349" };
+      # "test_soft_fail" string => "any",
+      #   meta => { "redmine6349" };
```

Comment:
The comment was to keep the link to the ticket. I'll add a "tags" meta variable instead.